packages/@ama-sdk/swagger-builder/src/core/formatters/json-formatter.ts

```diff
@@ -20,23 +19,23 @@ export class JsonFormatter implements Formatter {
   /** @inheritdoc */
   public async generate(spec: any): Promise<void> {
     const content = JSON.stringify(spec, null, 2);
-    await mkdirp(path.dirname(this.filePath));
-    const res = await new Promise<void>((resolve, reject) => fs.writeFile(this.filePath, content, (err) => err ? reject(err) : resolve()));
+    await fs.promises.mkdir(path.dirname(this.filePath), {recursive: true});
```

Comment:
Does it reject if the folder already exist?

Reply:
No, it won't reject when `recursive: true`